docs/content/install.sh

```diff
@@ -147,18 +150,32 @@ cd $unzip_dir/*
 
 case "$OS" in
   'linux')
+    # - - - - - - - - - - - - - - - - - - - -
     #binary
-    cp rclone /usr/bin/rclone.new
-    chmod 755 /usr/bin/rclone.new
-    chown root:root /usr/bin/rclone.new
-    mv /usr/bin/rclone.new /usr/bin/rclone
+    # First verify our sudo ability. If we pass that then execute the steps
```

Comment:
Do we need to sudo in here?

The instructions say

    sudo -v ; curl https://rclone.org/install.sh | sudo bash

So we should be root?

Reply:
Sorry, I obviously didn't do a good enough job putting the comments in.
The sudo is actually there. The critical parts that require root priv's occurs just after the snippet you showed it reads like this:
```
# First verify our sudo ability. If we pass that then execute the steps
# requested but make sure we're doing it as root and not as any other user.
# (If sudo was run before this, the user may be set to something *other*
# than 'root', this makes sure.
sudo -v
sudo -u root -- bash -c
```

The last two lines there are the enablers. First, verify (and authenticate) the ability to become root. then second line shown above says "Make sure you become root and then execute the following steps in the bash shell."

Should I go back and modify the comments to explain better?

Reply:
Sorry, I didn't make my point very clearly. My point was  that if you follow the instructions then the script will already be running as root, so there is no need for `sudo` at all is there?